themes/adminimal_medela/source/sass/components/_forms.scss

```diff
@@ -12,3 +12,9 @@
     }
   }
 }
+
+.node-form {
+  .hidden {
```

Comment:
@Tjocksmurf - is there a particular reason why the `hidden` classname is nested within `node-form`? If not, then it would be better not to nest it.

Reply:
@Benjen Only that I know if the class "hidden" is used somewhere else in the theme. If it is I don't want to hide something by misstake.